english/meditation/meditation.md

```diff
@@ -1,3 +1,5 @@
 You should relax, you completed the git course, good job!
+You feel relaxed and sleepy... Go to sleep?
 
-Click [here](https://www.youtube.com/watch?v=2u86NS3NtMw)!
\ No newline at end of file
+Click [here](https://www.youtube.com/watch?v=2u86NS3NtMw)!
+[Go to sleep](expired-milk/expired-milk.md)?
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.